Equilobe.DailyReport.SL/ReportService.cs

```diff
@@ -0,0 +1,84 @@
+﻿using Equilobe.DailyReport.DAL;
+using Equilobe.DailyReport.Models.Interfaces;
+using Equilobe.DailyReport.Models.Jira;
+using Equilobe.DailyReport.Models.ReportFrame;
+using Equilobe.DailyReport.Models.Storage;
+using System;
+using System.Linq;
+
+namespace Equilobe.DailyReport.SL
+{
+    public class ReportService : IReportService
+    {
+        public IJiraService JiraService { get; set; }
+
+        #region IReportService Implementation
+        public void UpdateDashboardData(long instanceId)
```

Comment:
This method is called UpdateDashboardData but I see that you update the users. Change the name of the method. Or if this method will call all the other methods that do updates, extract the users part in a separate method and call it in here.

Reply:
Initially, I thought about a method which updates the dashboard data, including users, worklogs, comments and so on. But yeah, you are right, the name of the method does not stand for what it does. I created a new method for what I was working on and called that method inside this one.